package/tests/test_server.py

```diff
@@ -298,11 +357,67 @@ def test_nodes_endpoint(cli_runner, client):
 @pytest.mark.usefixtures("patched_get_project_context")
 def test_pipeline_flag(cli_runner, client):
     """Test that running viz with `--pipeline` flag will return a correct pipeline."""
-    cli_runner.invoke(server.commands, ["viz", "--pipeline", "another"])
+    cli_runner.invoke(server.commands, ["viz", "--pipeline", "second"])
     response = client.get("/api/nodes.json")
     assert response.status_code == 200
     data = json.loads(response.data.decode())
-    assert data == {"edges": [], "layers": [], "nodes": [], "tags": []}
+    assert data == {
+        "edges": [
+            {"source": "7366ec9f", "target": "760f5b5e"},
+            {"source": "f1f1425b", "target": "760f5b5e"},
+            {"source": "760f5b5e", "target": "60e68b8e"},
+            {"source": "60e68b8e", "target": "24d754e7"},
+            {"source": "f1f1425b", "target": "24d754e7"},
+        ],
+        "layers": [],
+        "nodes": [
+            {
+                "full_name": "func",
+                "id": "760f5b5e",
+                "name": "Func",
+                "pipelines": ["second"],
+                "tags": [],
+                "type": "task",
+            },
+            {
+                "full_name": "func1",
+                "id": "24d754e7",
+                "name": "Func1",
+                "pipelines": ["second"],
+                "tags": [],
+                "type": "task",
+            },
+            {
+                "full_name": "bob_in",
+                "id": "7366ec9f",
+                "layer": None,
+                "name": "Bob In",
+                "pipelines": ["second"],
+                "tags": [],
+                "type": "data",
+            },
+            {
+                "full_name": "bob_out",
+                "id": "60e68b8e",
+                "layer": None,
+                "name": "Bob Out",
+                "pipelines": ["second"],
+                "tags": [],
+                "type": "data",
+            },
+            {
+                "full_name": "parameters",
+                "id": "f1f1425b",
+                "layer": None,
+                "name": "Parameters",
+                "pipelines": ["second"],
+                "tags": [],
+                "type": "parameters",
+            },
+        ],
+        "pipelines": [{"id": "second", "name": "Second"}],
```

Comment:
Is this the expected user experience? If there is a dropdown, when you launch kedro viz with a single pipeline, they should still be able to select the other pipeline in the dropdown right?